README.md

```diff
@@ -12,6 +12,8 @@ Eclipse Theia Blueprint is a template for building desktop-based products based
 </div>
 
 [![Installers](https://img.shields.io/badge/download-installers-blue.svg?style=flat-curved)](https://theia-ide.org/docs/blueprint_download/)
+[![Build Status](https://ci.eclipse.org/theia/buildStatus/icon?subject=Main&job=Theia2%2Fmaster)](https://ci.eclipse.org/theia/job/Theia2/job/master/)
```

Comment:
For my own understanding, why is this build called `Theia2`, would it not be better to rename it to something more meaningful like `theia-latest`?

Reply:
I think `Theia2` is just the internal Jenkins name for our main pipeline. It's a bit confusing, but it probably pre-dates us coming-up with the name `Blueprint`, and is not meant to be really user-facing - the `Display Name` is meant for that. It's still confusing because the display name is `Theia - master` but actually this build uses solid Theia revisions. So maybe `Blueprint stable` or something like that would be more descriptive. 

But in any case, these are Jenkins settings that end-up being used innocently by this PR here, so I see no reason to go into all this too deeply - we can change (some of) the Jenkins setting any time we want, and update any reference. 

![image](https://user-images.githubusercontent.com/25749063/145599438-ec32b5ab-c5fb-400f-a1c3-6772188d856d.png)
![image](https://user-images.githubusercontent.com/25749063/145599650-4ad7362c-8359-4b03-8edf-d25fac9b563b.png)